packages/common/src/compiler/translator.ts

```diff
@@ -115,13 +115,17 @@ const convertDimension = (
         sql = convertTimezone(sql, 'UTC', 'UTC', targetWarehouse);
     }
     const isIntervalBase =
-        timeInterval === undefined && isInterval(type, column);
+        timeInterval === undefined &&
+        isInterval(type, column) &&
+        !isAdditionalDimension;
```

Comment:
Additional dimensions can never be a interval base (this is what was causing the dimensions to not show in the frontend)